src/dso_api/dynamic_api/remote/views.py

```diff
@@ -125,6 +130,15 @@ def _call_remote(self, url="") -> Union[dict, list]:
         else:
             url = urljoin(self.endpoint_url, url)
 
+        if query_params:
+            url = urljoin(
+                self.endpoint_url,
+                "?" + "&".join(f"{k}={v}" for k, v in query_params.items()),
+            )
+
+        if "{table_id}" in url:
+            url = url.replace("{table_id}", self.table_id)
```

Comment:
Kan zonder if statement:
```pyhon
url = url.replace("{table_id}", self.table_id)
```
Immers, als `replace` de te vervangen string niet kan vinden retourneert deze gewoon de oorspronkelijke string. Dit is korter en voorkomt het twee keer doorlopen van de string.

Alternatief: Omdat `{table_id}` de [format string sytax](https://docs.python.org/3.8/library/string.html#formatstrings) volgt, zou je zelfs (en wederom zonder if statement):

```python
url = url.format(table_id=self.table_id)
```
kunnen doen. Doch dat loopt stuk als er nog meer _replacement fields_ in de string zitten.